src/Cameras/babylon.camera.ts

```diff
@@ -462,23 +462,6 @@
 
             this._refreshFrustumPlanes = true;
 
-            if (!this.parent || !this.parent.getWorldMatrix) {

```

Comment:
We need to build worldMatrix here

Reply:
Is worldMatrix necessary every frame?

Reply:
It is not every frame but every time we build the view

Reply:
I don't mean calculating it. I mean that getWorldMatrix is called every frame.

Reply:
We just need to make sure that camera._worldMatrix is updated when view matrix is updated

Reply:
I've already done that with my last commit. The way the code worked before it updated the world matrix to the inverse of the local view matrix as a temp matrix. The _worldMatrix only gets updated to the correct value when getWorldMatrix is called. Anyways, I think the current changes work the way you are describing.

Reply:
Oh ok... I'll check later then:) on my phone now